apps/web/src/pages/quick-start/steps/GetStarted.tsx

```diff
@@ -25,16 +27,27 @@ const ChannelsConfigurationHolder = styled.div`
 
 export function GetStarted() {
   const segment = useSegment();
+  const { currentOrganization } = useAuthContext();
   const [clickedChannel, setClickedChannel] = useState<{
     open: boolean;
     channelType?: ChannelTypeEnum;
   }>({ open: false });
 
+  const isNovuProd = !IS_DOCKER_HOSTED && ENV === 'production';
+  // open modal only for prod users
+  const isOnboardingModalEnabled = isNovuProd && localStorage.getItem('onboarding_modal') === 'true';
```

Comment:
why do we only show this for prod users? how we test it locally?

Reply:
onboarding workflow to work requires Novu provider enabled. In local machine, novu email provider is not possible to enable. I added this check to avoid that case. Only way to test in local machine is to remove this condition

Reply:
> novu email provider is not possible to enable

that's not true ;) I do have it enabled ;)